.github/workflows/NoNo.yml

```diff
@@ -0,0 +1,40 @@
+name: NoNo
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on pull request events but only for the main branch
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    env:
+      GREP_FOR: 'audio_play_sound'
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: fail
+        if: ${{ failure() }}
+        run: echo grep -r --exclude='___*' ${{ env.GREP_FOR }}
```

Comment:
does this also exclude internal code?

e.g. https://github.com/spacebake/Loungeware/blob/main/Loungeware/scripts/public_audio_functions/public_audio_functions.gml#L18